config/config.go

```diff
@@ -251,6 +255,11 @@ type SentryConfiguration struct {
 	DSN string `json:"dsn" envconfig:"CONVOY_SENTRY_DSN"`
 }
 
+type RetentionPolicyConfiguration struct {
```

Comment:
Didn't we have this before? Or was it only in the project settings?

Reply:
Only project settings

Reply:
ok